### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by jhlivingston**
Creating a dependabot config that uses the same logic as the config on Node-Disks.

Checks npm and github-actions daily for updates.  Kristen and Nico are default reviewers.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    versioning-strategy: increase
+    reviewers:
+      - "KristenKatona"
+      - "nicogonza"
+    ignore:
+      - dependency-name: "@types/node"
+        versions:
+          - ">= 10"
+  - package-ecosystem: "github-actions"
+     open-pull-requests-limit: 10
```
Comment (ColgateLeoAscenzi) [inline .github/dependabot.yml:19]: Same here, just need to remove the space quickly.

**Review by ColgateLeoAscenzi — CHANGES_REQUESTED** [review]
Same as with the node-disks yml, this looks good! Just fix the extra space and I think ask @KristenKatona

**Review by jhlivingston — COMMENTED** [review]
Change committed.

**Review by ColgateLeoAscenzi — APPROVED** [review]
👍

**Pull request merged**